.github/workflows/bumpversion.yaml

```diff
@@ -7,27 +7,64 @@ on:
 
 jobs:
   bump_version:
-    if: "!startsWith(github.event.head_commit.message, 'bump:')"
     runs-on: ubuntu-latest
     name: "Bump version and create changelog with commitizen"
+    env:
+      GIT_USER_NAME: github-actions
+      GIT_USER_EMAIL: github-actions@github.com
     steps:
-      - name: Check out
-        uses: actions/checkout@v2
+      - name: Check out code
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          token: "${{ secrets.GITHUB_TOKEN }}"
-      - id: cz
-        name: Create bump and changelog
-        uses: commitizen-tools/commitizen-action@master
+
+      - name: Install poetry
+        run: pipx install poetry
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4.5.0
+        with:
+          python-version: "3.9"
+          cache: "poetry"
+          cache-dependency-path: "poetry.lock"
+
+      - name: Install dependencies
+        run: |
+          poetry install --only=commit-lint --no-root
+
+      - name: Bump version
+        if: "!startsWith(github.event.head_commit.message, 'bump:')"
+        run: |
+          poetry run cz bump --files-only --yes --changelog --changelog-to-stdout > changelog-increment.md
+          git config user.name ${GIT_USER_NAME}
+          git config user.email ${GIT_USER_EMAIL}
+          echo "bump: release $(poetry run cz version --project)" > commit-message
+          echo "" >> commit-message
```

Comment:
Why the extra newline here?

Reply:
It's a common git convention. 

>Though not required, it’s a good idea to begin the commit message with a single short (less than 50 character) line summarizing the change, **followed by a blank line** and then a more thorough description.

Source: https://git-scm.com/docs/git-commit#_discussion (emphasis is mine)

I _THINK_ it's important for parsing commit body (`%b`) later

Reply:
Ohhhh, yes, I understand your intent now. That makes perfect sense and I completely agree! 👍 

In that case, you could just include the newline in the previous command like:

```
echo "bump: release $(poetry run cz version --project)\n" > commit-message
```

Though, one extra `echo` doesn't make much of a difference, and I don't mean to split hairs on micro-optimizations. 😛